make/test.mk

```diff
@@ -393,7 +393,7 @@ create-host-project:
 	-oc label ns --overwrite=true ${HOST_NS} app=host-operator
 
 .PHONY: create-host-resources
-create-host-resources: create-spaceprovisionerconfigs-for-members tiers-via-ksctl
+create-host-resources: create-spaceprovisionerconfigs-for-members tiers-via-ksctl tiers-go-templates
```

Comment:
Would `tiers-via-ksctl` be ever used without `tiers-go-templates`? If they're always used together, I'd maybe just merge them together.

Reply:
i had thought about that, but since we would remove this` tiers-via-kctl` during the clean up, i though about keeping the `tiers-go-template` seprate, so it becomes easier going forward.
WDYT?